### PR TITLE
Couple minor things in cmstestsuite (in particular for RunTimeTest

### PR DESCRIPTION
**Pull request by stefano-maggiolo**
This change is [Reviewable](https://reviewable.io/reviews/cms-dev/cms/822)

### CODE REVIEW

File: cmstestsuite/testrunner.py [hunk 1]
```diff
@@ -100,13 +101,24 @@ def load_cms_conf():
         return get_cms_config()
 
     def log_elapsed_time(self):
+        def duration_to_human(start, end):
+            millis = int((end - start).total_seconds() * 1000)
```
Comment (lerks) [inline cmstestsuite/testrunner.py]: Tip: you could use `str(end - start)` which returns a similar format (though not exactly identical) to save yourself some effort.

File: cmstestsuite/testrunner.py [hunk 2]
```diff
@@ -278,20 +278,29 @@ def _all_user_tests(self):
...
-    def submit_tests(self):
+    def submit_tests(self, concurrent_submit_and_eval=True):
         """Create the tasks, and submit for all languages in all tests.
 
+        concurrent_submit_and_eval (boolean): if False, stop ES while
+            we are submitting, so that we have a clearer view of the time
+            each step takes.
+
         """
+        def start_eval_services():
```
Comment (lerks) [inline cmstestsuite/testrunner.py]: There's a discrepancy between what this function does and what the docstring says it does.

File: cmstestsuite/testrunner.py [hunk 3]
```diff
@@ -278,20 +278,29 @@ def _all_user_tests(self):
...
+            self.ps.start("EvaluationService", contest=self.contest_id)
```
Comment (lerks) [inline cmstestsuite/testrunner.py]: Delaying just ES is enough to achieve what you wanted. You could start PS (and possibly the workers) together with CWS at the beginning. Or not, doesn't matter much. As you prefer.

File: cmstestsuite/testrunner.py (hunk 1 above)
Comment (stefano-maggiolo) [reply cmstestsuite/testrunner.py]: Ah, I was looking for something like that -,-

File: cmstestsuite/testrunner.py (hunk 2 above)
Comment (stefano-maggiolo) [reply cmstestsuite/testrunner.py]: Done

File: cmstestsuite/testrunner.py (hunk 3 above)
Comment (stefano-maggiolo) [reply cmstestsuite/testrunner.py]: Done

**Comment by lerks** [comment]
LGTM

**Comment by stefano-maggiolo** [comment]
Merged

**Pull request closed without merging**